common/src/main/java/com/revenuecat/purchases/common/errors.kt

```diff
@@ -40,7 +41,11 @@ enum class BackendErrorCode(val value: Int) {
 fun Exception.toPurchasesError(): PurchasesError {
     return when (this) {
         is JSONException, is IOException -> {
```

Comment:
Would this work like:
```suggestion
        is UnknownHostException -> {
          PurchasesError(PurchasesErrorCode.UnknownHostError, localizedMessage)
        }
        is JSONException, is IOException -> {
            PurchasesError(PurchasesErrorCode.NetworkError, localizedMessage)
        }
```